src/components/FormField/Select.tsx

```diff
@@ -0,0 +1,186 @@
+import React, { HTMLAttributes, ReactNode } from "react";
+import * as RadixSelect from "@radix-ui/react-select";
+import { Icon } from "../Icon/Icon";
+import {
+  Error,
+  FormElement,
+  FormRoot,
+  ItemSeparator,
+  Label,
+  MenuContent,
+  OptionContainer,
+  ScrollButton,
+} from "./commonElement";
+import { uniqueId } from "lodash";
+import styled from "styled-components";
+
+interface SelectProps {
+  placeholder?: string;
+  label: ReactNode;
+  children: ReactNode;
+  error?: ReactNode;
+}
+
+type Props = RadixSelect.SelectProps &
+  Omit<HTMLAttributes<HTMLDivElement>, "children" | "placeholder"> &
+  SelectProps;
+
+const SelectRoot = styled(RadixSelect.Root)`
+  width: 100%;
+`;
+const SelectTrigger = styled(RadixSelect.Trigger)<{ error: boolean }>`
+  width: 100%;
+  ${FormElement};
```

Comment:
let's remove these css classes. Our unit is a component. if we only work with components life is much easier rather than mixing components and css classes